### DISCUSSION TITLE
Technicolor DGA4132 with erased eripv2

### DISCUSSION THREAD

**Discussion by GabriPir1003** [hack-technicolor/hack-technicolor#295] [Q&A] [answered]
is there any way to recover it? I don't have any backup.

**Comment by LuKePicci** [comment, marked as answer]
Not really. Assuming you have a way to reliably rewrite the eripv2 partition via NAND programmer you would need another eripv2 partition from another 4132 and you need to patch it in order to match your own chip I'd. Maybe @smx-smx could remember us if chip I'd is a signed eripv2 entry or not.

**Comment by GabriPir1003** [comment]
Thanks anyway.
